src/components/button/button.major.stories.mdx

```diff
@@ -23,14 +23,14 @@ import Button from "carbon-react/lib/components/button";
 
 ## Examples
 
-### Primary Buttons
+### Major Primary Buttons
```

Comment:
Yes it was requested to separate button into two stories Major and Minor as per screenshot attached.

Reply:
Hi @DobroTora,

I am inclined to agree with @edleeks87 on this, as since we have a new separate `MinorButton` component, adding the "major" reference and scope in the stories for `Button` doesn't really convey very much and could be slightly confusing for new Carbon consumers. 

I do agree in the future we should rename `Button` to `MajorButton` so we are aligned with DS's naming convention as @LaiaSage mentioned, but since this would be a breaking change it would be better to release this with other breaking changes we need to make, so consumers won't have to do multiple Carbon upgrades for trivial changes.

Reply:
So to give context to @edleeks87 and @Parsium. Originally when Laia did the UXQA we had minor and major as one component and on one storybook page. We therefore suggested adding major and minor labels to each section would help differentiate  the two buttons.

If minor button is now a separate component, we can remove the reference to major to avoid dev confusion as @Parsium  suggests. We have an carbon/DS alignment stream in the new year where we can update naming then if required. Perhaps we fix all the names in one go as you suggest (all breaking changes).